### DISCUSSION TITLE
User changeable time axis

### DISCUSSION THREAD

**Discussion by MoBosse** [open-sdg/open-sdg#2134] [Ideas]
Hi all,
a colleague of a subnational statistical office wants to build an open sdg platform and wants to implement a function that allows the user to change the displayed years in chart and table. 
Does anyone know whether this has been considered before and whether there is a corresponding function in chat.js?

**Comment by brockfanning** [comment]
@MoBosse A quick idea that comes to mind is to use the "Series" column for this, like for example:

Year|Series|Value
---|---|---
2018|Before 2020|100
2019|Before 2020|110
2020|After 2019|120
2021|After 2019|130

**Comment by MoBosse** [comment]
@brockfanning yes, this is exactly the workaround I suggested as a quick solution. But I thought there might be a more elegant solution... Never mind if not though. I didn't have the feeling that it was too important.
